Product_Landing_Page.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Product_Landing_Page.css">
+    <title>Document</title>
+</head>
+<body>
+       <div>
+        <header id="header">
+            <img id="header-img"src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="Company Logo here">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a class="nav-link"href="index.html#section1">ABOUT</a></li>
+                    <li><a class="nav-link"href="#section2">APARTMENTS</a></li>
+                    <li><a class="nav-link"href="index.html#section3">LANDLORDS</a></li>
```

Comment:
You would do fine to arrange these lines properly, so that the codes would fall in their proper place, eg </li> between lines 17 and 18.